integration_tests/store/test_store_revisions.py

```diff
@@ -64,9 +62,10 @@ def test_revisions_with_login_bad_snap_with_arch(self):
             "Snap 'mysnap' for 'i386' was not found in '16' series.",
             str(error.output))
 
-    @unittest.skipUnless(
-        os.getenv('TEST_STORE', 'fake') == 'fake', 'Skip fake store.')
     def test_revisions_fake_store(self):
+        if not self.is_store_fake():
```

Comment:
if these 3 tests go into their on `FakeStoreRevisionsTestCase` (or whatever better name) you can just do this in `setUp`